client/components/CustomBankField.js

```diff
@@ -0,0 +1,66 @@
+import {has, translate as t} from '../Helpers';
+import T from './Translated';
+
+export default class CustomBankField extends React.Component {
+
+    constructor(props) {
+        has(props, 'params');
+
+        super(props);
+    }
+
+    getValue() {
+        let node = this.refs[this.props.params.name].getDOMNode();
+        return {
+            name: this.props.params.name,
+            value: (this.props.params.type === "number") ? parseInt(node.value, 10) : node.value
+        };
+    }
+
+    setValue(value) {
```

Comment:
setValue fixed.

I still have to retrieve the current value of each customField though to be able to display it…